recipes/gperftools/all/test_package/test_package_indirect.c

```diff
@@ -0,0 +1,9 @@
+#include <assert.h>
+#include <stdlib.h>
+#include <stdio.h>
+
+int main() {
+    void *p = malloc(100);
+    free(p);
```

Comment:
```suggestion
    printf("%p\n", p);    
    free(p);
```
Without this, there is a risk that the compiler choses to optimize this into `return EXIT_SUCCESS` : https://godbolt.org/z/vP3vq9o37 vs https://godbolt.org/z/GvTYEGPqv